dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf

```diff
@@ -1,151 +0,0 @@
-resource "kafka_topic" "eqdb_loader" {
```

Comment:
Just to be on the safe side: you don't need the data from these topics, right? As they'll be deleted

Reply:
We do not need it yep. The namespace is already gone.